app/controllers/characters_controller.rb

```diff
@@ -28,7 +29,7 @@ def create
 
     respond_to do |format|
       if @character.save
-        format.html { redirect_to @character, notice: 'Character was successfully created.' }
+        format.html { redirect_to characters_path, notice: 'Character was successfully created.' }
```

Comment:
Just curious what the thought is by redirecting to the index, rather than the character?